cmake/README.md

```diff
@@ -39,19 +39,21 @@ cmake <options> <repo_root>
 
 Some useful options:
 - `-G <generator>` - specify particular project generator. See `cmake --help` for details.
-- `-DCMAKE_BUILD_TYPE=Debug` - specify for Debug build. It is not applicable for multiconfig generators, e.g. for Visual Studio* generator.
+- `-DCMAKE_BUILD_TYPE=Debug` - specify for Debug build. It is not applicable for multi-config generators, e.g., Microsoft* Visual Studio* generator.
 
-#### TBBBind library configuration
+#### TBBBind Library Configuration
 
-The TBBbind library has three versions: `tbbbind`, `tbbbind_2_0`, and `tbbbind_2_5`. Each of these versions is linked with corresponding HWLOC library version: 
-- `tbbbind` links with HWLOC 1.11.x 
-- `tbbbind_2_0` links with HWLOC 2.1–2.4
-- `tbbbind_2_5` links with HWLOC 2.5 and later
+> **_TIP:_** It is recommended to install the HWLOC* library. See [oneTBB documentation](https://oneapi-src.github.io/oneTBB/GSG/next_steps.html#hybrid-cpu-and-numa-support) for details.
+
+The TBBbind library has three versions: `tbbbind`, `tbbbind_2_0`, and `tbbbind_2_5`. Each of these versions is linked with the corresponding HWLOC* library version: 
+- `tbbbind` links with `HWLOC 1.11.x`
+- `tbbbind_2_0` links with `HWLOC 2.1–2.4`
+- `tbbbind_2_5` links with `HWLOC 2.5` and later
 
 The search for a suitable version of the HWLOC library is enabled by default. If you want to use a specific version of the library, you can specify the path to it manually using the following CMake variables:
 
  - `CMAKE_HWLOC_<HWLOC_VER>_LIBRARY_PATH` - path to the corresponding HWLOC version shared library on Linux* OS or path to `.lib` file on Windows* OS
- - `CMAKE_HWLOC_<HWLOC_VER>_INCLUDE_PATH` - path to the corresponding HWLOC version including directory
+ - `CMAKE_HWLOC_<HWLOC_VER>_INCLUDE_PATH` - path to the corresponding HWLOC version, including directory
```

Comment:
Not sure of the right wording, but this is the path of the HWLOC/include directory, not of the HWLOC directory.

Reply:
thanks, fixed!